server/src/main/java/com/epam/aidial/core/server/validation/ListCollector.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.aidial.core.server.validation;
+
+import com.networknt.schema.Collector;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListCollector<T> implements Collector<List<T>> {
```

Comment:
Is the class thread-safe?

I don't see any case where it's concurrently used.

Reply:
It can be used in that manner setting corresponding options in validator so I decided to implement it thread safe.

Reply:
synchronization removed